fluent-bundle/src/bundle.rs

```diff
@@ -22,21 +22,42 @@ pub struct Message {
     pub attributes: HashMap<String, String>,
 }
 
-/// `FluentBundle` is a collection of localization messages which are meant to be used together
-/// in a single view, widget or any other UI abstraction.
+/// A collection of localization messages for a single locale, which are meant
+/// to be used together in a single view, widget or any other UI abstraction.
 ///
-/// # `FluentBundle` Life-cycle
+/// # Examples
 ///
-/// To create a bundle, call `FluentBundle::new` with a locale list that represents the best
-/// possible fallback chain for a given locale.  The simplest case is a one-locale list.
+/// ```
+/// use fluent_bundle::bundle::FluentBundle;
+/// use fluent_bundle::resource::FluentResource;
+/// use fluent_bundle::types::FluentValue;
+/// use std::collections::HashMap;
 ///
-/// Next, call `add_messages` one or more times, supplying translations in the FTL syntax. The
+/// let resource = FluentResource::try_new("intro = Welcome, { $name }.".to_string()).unwrap();
+/// let mut bundle = FluentBundle::new(&["en-US"]);
+/// bundle.add_resource(&resource);
+///
+/// let mut args = HashMap::new();
+/// args.insert("name", FluentValue::from("Rustacean"));
+///
+/// let value = bundle.format("intro", Some(&args));
+/// assert_eq!(value, Some(("Welcome, Rustacean.".to_string(), vec![])));
+///
+/// ```
+///
+/// # `FluentBundle` Life Cycle
+///
+/// To create a bundle, call [`FluentBundle::new`] with a locale list that represents the best
+/// possible fallback chain for a given locale. The simplest case is a one-locale list.
+///
+/// Next, call [`add_resource`] one or more times, supplying translations in the FTL syntax. The
 /// `FluentBundle` instance is now ready to be used for localization.
 ///
 /// To format a translation, call `get_message` to retrieve a `fluent_bundle::bundle::Message` structure
 /// and then `format` it within the bundle.
```

Comment:
That requires updating too. We don't have `get_message` anymore, and you just pass a string id to `format`/`format_message` or pass `id.attr` path to `format.